--- /opt/pidp11/install/install.sh	2025-07-31 13:10:19.956532329 -0700
+++ unattended_install.sh	2025-07-31 13:21:07.353373899 -0700
@@ -39,7 +39,8 @@
 
 while true; do
     echo
-    read -p "Set required access privileges to pidp11 simulator? " yn
+    #read -p "Set required access privileges to pidp11 simulator? " yn
+    yn=Y
     case $yn in
         [Yy]* )
             # make sure that the directory does not have root ownership
@@ -69,7 +70,8 @@
 # =============================================================================
 while true; do
     echo
-    read -p "Install the required dependencies? " prxn
+    #read -p "Install the required dependencies? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             sudo apt update
@@ -96,7 +98,8 @@
 # Configure PCAP interface so it can be used without sudo
 
 echo
-read -p "Configure PCAP permissions for current user? " prxn
+#read -p "Configure PCAP permissions for current user? " prxn
+prxn=Y
 case $prxn in
     [Yy]* ) 
 	    # Set permissions for client11 to be able to access the lobpcap interface in simh without sudo
@@ -116,7 +119,8 @@
 
 while true; do
     echo
-    read -p "(Y) to install precompiled binaries, or (C)ompile from source, or (S)kip? " prxn
+    #read -p "(Y) to install precompiled binaries, or (C)ompile from source, or (S)kip? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             # Query the system architecture
@@ -170,7 +174,8 @@
 # =============================================================================
 while true; do
     echo
-    read -p "Install PiDP-11 package into OS? " prxn
+    #read -p "Install PiDP-11 package into OS? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             # setup 'pdp.sh' (script to return to screen with pidp11) 
@@ -202,7 +207,8 @@
                 echo
                 echo
                 echo "Autostart the PDP-11 using the GUI(Y) or .profile (H)?"
-                    read -p "-- Y recommended, H is for headless Pis without GUI:" yhn
+                    #read -p "-- Y recommended, H is for headless Pis without GUI:" yhn
+                    yhn=Y
                 case $yhn in
                       [Yy]* ) 
                         mkdir -p ~/.config/autostart
@@ -245,7 +251,8 @@
 # =============================================================================
 while true; do
     echo
-    read -p "Download and install the PDP-11 operating systems? " prxn
+    #read -p "Download and install the PDP-11 operating systems? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             cd /opt/pidp11
@@ -268,7 +275,8 @@
 # =============================================================================
 while true; do
     echo
-    read -p "Add VT-52 desktop icon and desktop settings? " prxn
+    #read -p "Add VT-52 desktop icon and desktop settings? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             cp /opt/pidp11/install/vt52.desktop /home/pi/Desktop/
@@ -322,7 +330,8 @@
 # =============================================================================
 while true; do
     echo
-    read -p "2024 update: Add Chase Covello's updated 2.11BSD ? " prxn
+    #read -p "2024 update: Add Chase Covello's updated 2.11BSD ? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             # Directory path
@@ -400,7 +409,8 @@
 # =============================================================================
 while true; do
     echo
-    read -p "2024: Add Johnny Billquist's latest RSX-11MPlus with BQTC/IP? " prxn
+    #read -p "2024: Add Johnny Billquist's latest RSX-11MPlus with BQTC/IP? " prxn
+    prxn=Y
     case $prxn in
         [Yy]* ) 
             # Directory path
@@ -424,7 +434,8 @@
             echo "Files will be downloaded by anonymous ftp from dfupdate.se."
             echo "As a courtesy, leave your email address (that is ftp etiquette)"
             echo
-            read -p "Enter your email address: " email
+            #read -p "Enter your email address: " email
+            email="nospam@example.com"
             ftp_url="ftp://ftp.dfupdate.se/pub/pdp11/rsx/pidp"
             files=("pidp.dsk.gz" "pidp.tap.gz")
             cd ${dir}
